src/js/share/utils.js

```diff
@@ -268,4 +243,58 @@ export const generateID = (string) => {
  */
 export const escapeURI = (string) => {
   return encodeURI(string).replace('#', '%23');
+};
+
+/**
+ * render json object to workspace
+ * @param {object} jsonObject current scriptList as object
+ * @param {object} workspace where blocks are rendered
+ */
+export const jsonDomToWorkspace = (jsonObject, workspace) => {
+  const blockList = [];
+  blockList.push(jsonObject);
+  renderAndConnectBlocksInList(null, blockList, brickListTypes.noBrickList, workspace);
+};
+
+/**
+ * Renders and connects all blocks in a brickList, elseBrickList and loopOrIfBrickList.
+ * Function is calling itself if a *brickList is in a *brickList.
+ * @param {Object} parentBlock parent block to connect
+ * @param {Object} brickList, elseBrickList or loopOrIfBrickList
+ * @param {object} brickListType of list
+ * @param {object} workspace where blocks are rendered
+ */
+export const renderAndConnectBlocksInList = (parentBlock, brickList, brickListType, workspace) => {
```

Comment:
Here the main functionality of this ticket happens:

- render new block with its parameters/formValues
- check if block has a connection with previous blocks
- function call itself for every block in brickList, elseBrickList and loopOrIfBrickList